test1.js

```diff
@@ -3,3 +3,10 @@ var foo = '' ;
 if(foo  === "bar") {
   console.log( 'hey' )
 }
+
+
+var foobar = '' ;
```

Comment:
Codacy found an issue: [Strings must use doublequote.](https://app.dev.codacy.org/gh/machadoit-test-org/eslint-example/pullRequest?prid=26122)
```suggestion
var foobar = "" ;
```